csrc/evaluator_common.cpp

```diff
@@ -165,12 +165,13 @@ void PrecomputedValues::bindInputs(const KernelArgumentHolder& args) {
   for (const auto i : c10::irange((int64_t)inputs.size())) {
     const auto input = inputs[i];
     TORCH_INTERNAL_ASSERT(input != nullptr);
-    bindValue(input->evaluatorIndex(), *args[i]);
     if (auto tensor_input = dynamic_cast<TensorView*>(input)) {
       const auto& tensor = args[i]->as<at::Tensor>();
       if (!tensor.is_cpu()) {
         bindTensorMetaData(tensor_input, tensor);
```

Comment:
Because `GetMetaData::evaluate` needs an `ExpressionEvaluator` to be able to infer allocation domain sizes and strides, but in the context of `PrecomputedValues`, we do not have an instance of `ExpressionEvaluator`, so unfortunately we have to manually bind tensor metadata, instead of letting `GetMetaData::evaluate` do it.